.coveragerc

```diff
@@ -0,0 +1,6 @@
+# .coveragerc to control coverage.py
+[run]
+omit =
+    # omit anything in a tests directory
+    */migrations/*
+    */integration/*
```

Comment:
@billyhunt Why only cover integration tests? Why not omit the whole /tests/ directory?

Reply:
I tried that, and it lowered our test coverage! I could do it, but we would need to up the coverage in other places.  We might consider doing it moving forward if we improve the rest of it.